src/libs/ajax/DataRepo.ts

```diff
@@ -207,6 +207,10 @@ export type SnapshotAccessRequest = {
 };
 
 export interface DataRepoContract {
+  admin: () => {
+    adminRetrieveSnapshot: (snapshotId: string) => Promise<Snapshot>;
+    adminRetrieveDataset: (datasetId: string) => Promise<SnapshotDataset>;
```

Comment:
You're right, I don't think these types are correct. We don't have a type yet for TDR dataset and `Snapshot` doesn't include all the fields that the API returns. Since the caller doesn't need a typed value, and these APIs are only used for the support page, I think it's OK to use the support return type `SupportSummary` for these APIs.

Reply:
I think Phil is right about this - the idea is that these APIs would return any kind of JSON that the underlying service provided.